addons/account/tests/test_account_tax.py

```diff
@@ -71,57 +71,3 @@ def test_tax_is_used_when_in_transactions(self):
         })
         tax_reconciliation.invalidate_model(fnames=['is_used'])
         self.assertTrue(tax_reconciliation.is_used)
-
-    def test_tax_is_used_restrictions(self):
```

Comment:
Adding a new test to replace the old one would actually be even nicer :nerd_face: